package/yast2-network.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Mon Nov 26 11:57:35 UTC 2018 - mfilka@suse.com
+
+- bnc#934055
```

Comment:
What about referencing also this bug that is for SLES

Reply:
well i don't have that many bug numbers to choose from.

Reply:
Sorry, I forgot to paste the bug number :)

Reply:
https://bugzilla.suse.com/show_bug.cgi?id=972463